Hard mode in progress

nickywongdong opened the pull request:
Can i get some constructive criticism on what I've got so far?

nickywongdong committed:
merge from group

nickywongdong committed:
hard mode implmentation in progress

dmkato commented:
Yea, I mean the logic looks sound, but we're gonna have to find a way to make it object-oriented. Like I think we have to make a new class classes hard mode and change the computer place ship function from there? But I'm not 100% sure...

dmkato commented:
Ok I definitely didn't see the new stuff when I made that last comment, but let me take a closer look tomorrow. I'm like about to go to bed haha

jakevolkman committed:
Code Review

I like what you've done with the place.
We should test it soon.
When the classes get updated to follow the object oriented version in the rubric, there may be a lot of changes here.
Also, does this currently run?

jakevolkman commented:
I looked it over and I liked the methods you used. I haven't checked if it works yet but I might in the morning.
**Mr. Nelson: This is my code review. I wasn't aware of the review tool so I just added comments**

jakevolkman committed:
Additional Code Review

nickywongdong committed:
revised some logic with loops, and renamed variables up & down to fit with rest of code

nickywongdong committed:
constructive criticism noted

nickywongdong committed:
finalized hard mode with inheritance
